tests/test_icml_conference.py

```diff
@@ -3083,6 +3085,224 @@ def test_delete_assignments(self, openreview_client, helpers):
+        ]
+        assert submissions[1].readers == [
+            "ICML.cc/2023/Conference",
+            "ICML.cc/2023/Conference/Submission2/Senior_Area_Chairs",
+            "ICML.cc/2023/Conference/Submission2/Area_Chairs",
+            "ICML.cc/2023/Conference/Submission2/Reviewers",
+            "ICML.cc/2023/Conference/Submission2/Authors"        ]
+        assert submissions[4].readers == [
+            "ICML.cc/2023/Conference",
+            "ICML.cc/2023/Conference/Submission5/Senior_Area_Chairs",
+            "ICML.cc/2023/Conference/Submission5/Area_Chairs",
+            "ICML.cc/2023/Conference/Submission5/Reviewers",
+            "ICML.cc/2023/Conference/Submission5/Authors",
+            "ICML.cc/2023/Conference/Submission5/Ethics_Reviewers"
+        ]
+
+        reviews = openreview_client.get_notes(invitation='ICML.cc/2023/Conference/Submission1/-/Official_Review')
+        assert reviews and len(reviews) == 2
+        for review in reviews:
+            assert 'ICML.cc/2023/Conference/Submission1/Ethics_Reviewers' in review.readers
+
+        invitations = openreview_client.get_invitations(invitation='ICML.cc/2023/Conference/-/Ethics_Review')
+        assert len(invitations) == 2
+        invitation = openreview_client.get_invitations(id='ICML.cc/2023/Conference/Submission1/-/Ethics_Review')[0]
+        assert invitation
+        assert 'ICML.cc/2023/Conference/Submission1/Ethics_Reviewers' in invitation.invitees
+
+        # re-run ethics review stage
+        now = datetime.datetime.utcnow()
+        start_date = now - datetime.timedelta(days=1)
+        stage_note = pc_client.post_note(openreview.Note(
+            content={
+                'ethics_review_start_date': start_date.strftime('%Y/%m/%d'),
+                'ethics_review_deadline': due_date.strftime('%Y/%m/%d'),
+                'make_ethics_reviews_public': 'No, ethics reviews should NOT be revealed publicly when they are posted',
+                'release_ethics_reviews_to_authors': "No, ethics reviews should NOT be revealed when they are posted to the paper\'s authors",
+                'release_ethics_reviews_to_reviewers': 'Ethics Review should not be revealed to any reviewer, except to the author of the ethics review',
+                'remove_ethics_review_form_options': 'ethics_review',
+                'additional_ethics_review_form_options': {
+                    "ethics_concerns": {
+                        'order': 1,
+                        'description': 'Briefly summarize the ethics concerns.',
+                        'value': {
+                            'param': {
+                                'type': 'string',
+                                'maxLength': 200000,
+                                'markdown': True,
+                                'input': 'textarea'
+                            }
+                        }
+                    }
+                },
+                'ethics_review_submissions': '1,5,6,7,8,100',
```

Comment:
I think we should remove this field in the API 2 venues. can you try flagging the papers posting the note edit with the flag?

Reply:
This was added to the test already. If this is how we'd rather do it, I can remove this field from the ethics stage